nix/sources.json

```diff
@@ -24,16 +36,41 @@
         "type": "tarball",
         "url": "https://github.com/justinwoo/easy-purescript-nix/archive/d4879bfd2b595d7fbd37da1a7bea5d0361975eb3.tar.gz"
     },
+    "ghcide": {
```

Comment:
What are these forks? Are they getting upstreamed?

Reply:
So the only significant change I have made to the mpickering's object-code branch is to update it to use the patched version of `hie-bios` with support for the `hie.project` file.  I'm still not sure if the `hie.project` file is the right good solution.  Fixing the master branch to work with plutus (if it is possible) might make it unnecessary.

Reply:
Hm. I'm also not sure it's the right approach (see my earlier comment about inconsistent ghc-options). What if we just advised people to set it in `cabal.project.local` when they're using their IDE? Not ideal, admittedly, but given that we're still working on this I think that's acceptable - especially if in the long run we don't think we're going to need this because we'll fix the tools to work with plugins.

Reply:
Then there is the risk of `cabal test` using the wrong ghc-options.

Reply:
Yes. I'm suggesting we accept that.